src/main/java/com/github/kaktushose/jda/commands/internal/CommandDispatcher.java

```diff
@@ -123,6 +124,10 @@ public void onGuildMessageReceived(@Nonnull GuildMessageReceivedEvent event) {
         Optional<CommandCallable> command = commandMapper.findCommand(commands, input, settings.isIgnoreLabelCase());
         if (!command.isPresent()) {
             log.debug("No command for input {} found", Arrays.toString(input));
+
```

Comment:
no reason to null check these objects

Reply:
Was only used for debugging when discovering the embedFactory  NullPointerException. Just missed it, while cleaning the code.